src/main/resources/db/changelog/changes/13-create-categories-table.yaml

```diff
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-categories-table
+      author: mateacademystudent
+      changes:
+        - createTable:
+            tableName: categories
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
It's better to have explicit names for constraints like: primary/foreign key, unique